serve

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+_ALL="en_US.UTF-8" \
+bundle exec jekyll serve \
```

Comment:
We've been using `make run` to run our server, can we add the options you want to that script instead?